script.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function(){
+    const movieGluAPI = '5zwPt8bzCu7r7rifnKoCd827XemsTLcO6rk51vTy';
+    // const NYTMovie = '';//we will need to grab the movie name from wherever it will be rendered on the page
+    // const NTYQueryMovies = 'https://api.nytimes.com/svc/movies/v2/reviews/search.json?query='+ NYTMovie + 'api-key=' + NYTAPI;
+    const TMDBAPI = '0adfd846cf8f1168484b5da4e5339d7c';
```

Comment:
For our variables it would probably be best to keep the camel case standard for consistency.

Reply:
Same with all variables in this PR

Reply:
That makes it way cleaner! Sweet